### PR TITLE
New serverless pattern - Amazon SQS to CloudWatch Log group via Amazon EventBridge Pipes filter and transformation

### PR DESCRIPTION
**Pull request by anuprajpara**
*Issue #, if available:*

*Description of changes:*
This sample project demonstrates how to use Amazon EventBridge Pipes to connect an Amazon SQS queue with an AWS CloudWatch Log group. The pipe will apply a filter and transformation before sending the message to AWS CloudWatch Log group.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: eventbridge-pipes-sqs-to-cwlog/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+ aws sqs send-message \
+ --queue-url=SQS_URL \
+ --message-body '{"id":"789", "status": "REJECTED"}'
+
+ aws sqs send-message \
+ --queue-url=SQS_URL \
+ --message-body '{"id":"110", "status": "DELIVERED"}'
+
+Validate the result by reviewing the target AWS CloudWatch Log group.
```
Comment (ellisms) [inline eventbridge-pipes-sqs-to-cwlog/README.md:61]:
```suggestion
Validate the result by reviewing the target CloudWatch Log group.
```

**Review by ellisms — CHANGES_REQUESTED** [review]
Thanks for the submission, @anuprajpara . Requesting a few minor changes.

**Comment by anuprajpara** [comment]
Thanks @ellisms for the review comments.
Made the changes as per the review comments. Please review when you get a chance.

File: eventbridge-pipes-sqs-to-cwlog/template.yaml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+              - Effect: Allow
+                Action:
+                  - 'logs:CreateLogStream'
+                  - 'logs:PutLogEvents'
+                Resource: !GetAtt TargetCWLogGroup.Arn
+
+Outputs:
+  SourceQueueURL:
+    Description: "Source Queue URL"
+    Value: !GetAtt SourceQueue.Arn
```
[the remark this answers is not in the document]
Comment (ellisms) [reply eventbridge-pipes-sqs-to-cwlog/template.yaml:70]: @anuprajpara Was this updated? I don't see it.

[the remark this answers is not in the document]
Comment (anuprajpara) [reply eventbridge-pipes-sqs-to-cwlog/template.yaml]: Sorry. Forgot to add a modified template file.
Updated template file is now checked in.

**Review by ellisms — APPROVED** [review]

**Comment by ellisms** [comment]
Thanks. I'll add this to the publishing queue.

**Pull request merged**